model/src/exec_env.rs

```diff
@@ -180,4 +181,11 @@ impl<T: Env> EnvHandler<T> {
     pub fn block_number(&self) -> T::BlockNumber {
         T::block_number()
     }
+
```

Comment:
Missing doc comment.
Would you also be so kind to add a doc-comment to `block_number` above?